### ISSUE TITLE
in/decreased DNA replication

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3410]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

1. increased DNA replication
2. decreased DNA replication

>2. Synonyms (e.g., absent spleen)

1. more DNA replication; elevated DNA replication; accelerated DNA replication
2. less DNA replication; reduced DNA replication;  decelerated DNA replication

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1.  any anomaly in the chemical reactions and pathways resulting an increase in new strands of DNA being synthesized
2.  any anomaly in the chemical reactions and pathways resulting in a decrease in new strands of DNA being synthesized

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

PMID:29548889

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

abnormal DNA replication (MP:0008057)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0002-9553-7227
